homeassistant/components/ziggo_next/config_flow.py

```diff
@@ -0,0 +1,69 @@
+"""Config flow for Ziggo Next integration."""
+from collections import OrderedDict
+import logging
+
+import voluptuous as vol
+from ziggonext import ZiggoNext, ZiggoNextAuthenticationError, ZiggoNextConnectionError
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .const import CONF_COUNTRY_CODE, DOMAIN  # pylint: disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def validate_input(hass: core.HomeAssistant, data):
+    """Validate the user input allows us to connect.
+
+    Data has the keys from DATA_SCHEMA with values provided by the user.
+    """
+    api = ZiggoNext(data[CONF_USERNAME], data[CONF_PASSWORD], data[CONF_COUNTRY_CODE])
+    try:
+        api.initialize(_LOGGER)
+    except ZiggoNextConnectionError as ex:
+        raise CannotConnect(ex)
+    except ZiggoNextAuthenticationError as ex:
+        raise InvalidAuth(ex)
```

Comment:
How about simply catching these excepting inside `async_step_user` to avoid the need for two new exception types?